Microsoft.Dynamics365.UIAutomation.Sample/AzureDevOps/EasyRepro.runsettings

```diff
@@ -40,12 +40,12 @@
   </DataCollectionRunSettings>
   <!-- Parameters used by tests at runtime -->
   <TestRunParameters>
-    <Parameter name="OnlineUsername" value="" />
-    <Parameter name="OnlinePassword" value="" />
-    <Parameter name="OnlineCrmUrl" value="" />
+    <Parameter name="OnlineUsername" value="admin@CRM446560.onmicrosoft.com" />
```

Comment:
Please remove your credentials

Reply:
Appreciate the comment on this one.   These environment details for for a temp environment has been been removed.   We are working to get this commit removed as well.